### ISSUE TITLE
syntax highlighting not enabled

### ISSUE THREAD

**Issue by norregaarden-scalgo** [stevepryde/vscode-def-view#1]
Hi. Thanks for a great extension that complements "docs view". When the syntax highlighting works, it's great, but for most of my .ts/.tsx files, it's all white and without linebreaks. I can't seem to find a pattern for which files it works for and which it don't.

Edit: I want to add three things:
- The syntax highlighting either works or don't for the whole active file in the editor, regardless of where the displayed definition is imported from.
- When the syntax highlighting works, there is a horizontal scrollbar. I would prefer it if the lines just wrap, like they do in "docs view" and the terminal.
- It seems to work for all .css and .js files though

**Comment by stevepryde** [comment]
Yeah I've noticed this for some cases as well. I copied the syntax highlighting code from docs view and adapted it, so I'm not really sure how to resolve it. If you're able to provide some examples that might help. If you're able to contribute to this that would be really appreciated too. This is the first extension I created so a lot of it felt quite experimental 😅 It works most of the time but there's a lot of awkward handling to try to make it work in most cases.

With Rust (my main language) it gives me a start and end line that encompass the whole function. But most other languages only provide the start/end line for the function signature. So the extension tries to figure out where the function docs start and where the function ends using some hacky heuristics. It's not perfect but felt "good enough" for a v0.1.

Regarding the horizontal scroll - I added this intentionally for my own preference but it's easy to make it wrap as well. I haven't looked into adding settings yet. There is a setting inherited from docs view but I don't think it works unfortunately.

I don't have a lot of time to work on this atm sorry. Something specific like adding an option to toggle the horizontal scrollbar might be easy to do. If you do figure out why syntax highlighting doesn't work for some files, I'd be interested to have a look.

**Comment by norregaarden-scalgo** [comment]
A hint that might help you debug: If the definition is a React component, your view renders HTML as HTML, i.e. a >button< becomes an actual button.
I maybe would like to help if possible, if you can guide me on where to start...

Edit: Apparently github also renders HTML as HTML, hence the inverted ><
